multi-node/aws/pkg/config/default_stack_template.go

```diff
@@ -94,6 +94,28 @@ var defaultStackTemplate = `
         ""
       ]
     },
+    "ControllerPublic": {
+      "Fn::Equals": [
+        "{{.ControllerPublic}}",
+        "true"
+      ]
+    },
+		"CreateVPCResources": {
+			"Fn::Or": [
+				{
+					"Fn::Equals": [
+						"{{.VPCID}}",
+						""
+					]
+				},
+				{
+					"Fn::Equals": [
+						"{{.SubnetID}}",
+						""
+					]
+				}
+			]
+		},
```

Comment:
This should this be split up into CreateVPCResources and CreateSubnetResources? As of now, specifying a VPCID but not a SubnetID would be an invalid configuration (and not detected as such until the cloudformation template is submitted to the API). 

In the case that VPCID is specified but not SubnetID, a new subnet should be created in the existing VPC.